### PR TITLE
[d3-cloud] Support import syntax

### PR DESCRIPTION
**Pull request by oBusk**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/jasondavies/d3-cloud/blob/v1.2.5/index.js#L10

### CODE REVIEW

**Review by hansrwindhoff — APPROVED** [review]
LGTM,
sorry about the late review, missed the first mail...

File: types/d3-cloud/index.d.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@
 
 import * as d3 from 'd3';
 
+export default d3.layout.cloud;
```
Comment (PranavSenthilnathan) [inline types/d3-cloud/index.d.ts]: It seems like the [underlying library](https://github.com/jasondavies/d3-cloud/blob/master/index.js#L10) is not using `export default`, so this .d.ts file should not either. Note that if you want to use `import name from 'package';` in your consuming code, you can use the `esModuleInterop` compiler setting.

**Review by PranavSenthilnathan — CHANGES_REQUESTED** [review]

File: types/d3-cloud/index.d.ts (hunk 1 above)
Comment (oBusk) [reply types/d3-cloud/index.d.ts]: So the export should be like this?

```suggestion
export = d3.layout.cloud;
```

**Comment by oBusk** [comment]
🔔 @PranavSenthilnathan Does the changes look good?

**Comment by oBusk** [comment]
@RyanCavanaugh @andy-ms This PR has gotten stuck because @PranavSenthilnathan has requested changes and then not reviewed them. What process to move forward?

**Pull request merged**